.github/actions/macos_install/action.yml

```diff
@@ -6,7 +6,7 @@ runs:
     - name: Install MPI, OpenMP
       run: |
         brew install open-mpi
-        brew install libomp
+        brew reinstall libomp
```

Comment:
I made this change, but it didn't fix the problem. It seems that the problem has been fixed upstream though, so this change can be reverted

Reply:
Done in [9ea3abf](https://github.com/pyccel/pyccel/pull/1236/commits/9ea3abfa46b3d032e2791954f51364c4f7cb8c54).